### ISSUE TITLE
 The reconstruction error

### ISSUE THREAD

**Issue by Nate1874** [AllenCellModeling/torch_integrated_cell#7]
In your work, the reconstruction error is measured by pixel-wise binary crossentropy loss, which is BCECriterion in Torch. However, the in the torch documentation for this loss function, https://github.com/torch/nn/blob/master/doc/criterion.md#nn.BCECriterion,  the output can be between 0 and 1 but the target t[i] should be either 0 or 1.

The target of image reconstruction error is the original image X^r or X^r,s, but it has discrete values between 0 and 1. So why the binary crossentropy loss can work for reconstruction error?

Thanks.

**Comment by gregjohnso** [comment]
We use the BCE to make the error terms on the same scale with the adversaries. In practice, many other error terms will work well (MSE, L1, whatever).
